### ISSUE TITLE
Sidebearings Measurement needs check for italicAngle `None` value

### ISSUE THREAD

**Issue by colinmford** [RafalBuchner/StemPlow#12]
When turning on the "Measure against Sidebearings" option and pressing "f" I get the following traceback:

```
Observer '<stemPlow.StemPlowSubscriber.StemPlowSubscriber object at 0x13d53a900>' with '_roboFontCallback' failed: must be real number, not NoneType
Traceback (most recent call last):
  File "lib/eventTools/eventManager.pyc", line 165, in callObserver_withMethod_forEvent_withInfo_
  File "/Applications/RoboFont.app/Contents/Resources/lib/python3.12/mojo/subscriber.py", line 1718, in _roboFontCallback
  File "/Applications/RoboFont.app/Contents/Resources/lib/python3.12/mojo/subscriber.py", line 3421, in start
  File "/Applications/RoboFont.app/Contents/Resources/lib/python3.12/mojo/subscriber.py", line 3438, in fire
  File "/Applications/RoboFont.app/Contents/Resources/lib/python3.12/mojo/subscriber.py", line 1736, in _roboFontCoalescerCallback
  File "/Users/colinmford/Library/Application Support/RoboFont/plugins/StemPlow.roboFontExt/lib/stemPlow/StemPlowSubscriber.py", line 512, in glyphEditorDidMouseMove
  File "/Users/colinmford/Library/Application Support/RoboFont/plugins/StemPlow.roboFontExt/lib/stemPlow/StemPlowSubscriber.py", line 844, in getThicknessData
  File "/Users/colinmford/Library/Application Support/RoboFont/plugins/StemPlow.roboFontExt/lib/stemPlow/StemPlowSubscriber.py", line 148, in copyGlyphWithSidebearings
TypeError: must be real number, not NoneType
```
It is a normal glyph with outlines, it is not a blank glyph nor one with components.

Disabling the "Measure against Sidebearings" option returns the extension to normal function.

**Comment by colinmford** [comment]
The line in question is here:

https://github.com/RafalBuchner/StemPlow/blob/8a1dd2db57bf95c79f5942ef76e7b94ed7a4c127/source/code/stemPlow/StemPlowSubscriber.py#L148

My font doesn't have an italic angle (not zero, it's just blank). Typically not an issue to just leave blank, particularly in a new font, but probably the code should have a check for None and just return zero.

**Comment by RafalBuchner** [comment]
ha, sorry for making mess, I've fixed it with version 1.5.2
